manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -387,6 +387,7 @@ spec:
             - "--leader-election-renew-deadline=60s"
             - "--leader-election-retry-period=30s"
             - "--default-fstype=ext4"
+            - "--feature-gates=Topology=false"
             # needed only for topology aware setup
             #- "--feature-gates=Topology=true"
```

Comment:
Do we need to remove this commented line ?

cc: @shalini-b

Reply:
there is issues while provisioning volume without set Topology=false

Reply:
Correct, we need the flag = false, but we can remove the flag=true. Or add a comment to instruct the users deploying k8s clusters with topology=true.

Reply:
Let's keep the provisioner in-sync with TKGs YAMLs for now